app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -341,6 +345,7 @@ class ChatActivity : AppCompatActivity(), IChatView {
             chatPresenter.check(true)
             btnSpeak.setImageResource(R.drawable.ic_mic_24dp)
             btnSpeak.setOnClickListener({
+                btnSpeak.isClickable = false
```

Comment:
Should we disable the button instead?

Reply:
both are kinda similar but disabled seems more intuitive, will change that.